### PR TITLE
Feature/object creation

### PR DESCRIPTION
**Pull request by MDCurrent**
gonna  open a pr to comment on

### CODE REVIEW

File: src/main/java/main/java/Classes/MedicalRecord.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    
+    public MedicalRecord(){
+        super();
+    }
+
+    public int getRecordId(){
+        return recordId;
+    }
+    
+    public int getpatientId(){
```
Comment (MDCurrent) [inline src/main/java/main/java/Classes/MedicalRecord.java]: I think we should have a member variable, get, and set on doctorID too

File: src/main/java/main/java/Classes/MedicalRecordTypes/Medication.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+    public String getFrequency() {
+        return frequency;
+    }
+
+    public void setFrequency(String frequency) {
+        this.frequency = frequency;
+    }
+
+    public int getDosage() {
```
Comment (MDCurrent) [inline src/main/java/main/java/Classes/MedicalRecordTypes/Medication.java]: like the dosage I took that for the fabric struct

File: src/main/java/main/java/Classes/User.java [hunk 3]
```diff
@@ -7,30 +7,43 @@
...
 	@Column(name="password")
 	private String password;
 	@Column(name="firstname")
-	private String firstName;
+	private String firstname;
 	@Column(name="lastname")
-	private String lastName;
+	private String lastname;
+        @Column(name="DOB")
+        private Date DOB;
```
Comment (MDCurrent) [inline src/main/java/main/java/Classes/User.java]: we might need Date of Death if the bool isAlive is false

**Comment by Sean-Brogan** [comment]
already have one open if you want to use that one

**Pull request merged**